src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -33,6 +35,20 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses {@code roomNumber} into an {@code Integer} and returns it. Leading and trailing whitespaces will be
+     * trimmed.
+     * @throws ParseException if the specified roomNumber is invalid (not non-zero unsigned integer).
+     */
+    public static Integer parseRoomNumber(String roomNumber) throws ParseException {
```

Comment:
I understand it may be easier to return an `Integer` but this method is similar to the `parseIndex`. We can use the `Index` class provided in the original codebase and use `parseIndex` instead.

That was what I did in `AddTaskCommandParser` 😆

Reply:
cuz i feel like its not really an index, so grouping it under the index class seems wrong

Reply:
Okay, I see what you mean. For now, we are treating `RoomNumber` as an index but in the future we may allow the user to set their own room numbers.

Would it be convenient for you to move the room-related methods to `RoomParserUtil`?

If not, I could do that later because I intend to make `ParserUtil` an abstract class.

Reply:
sure i think i can do that

Reply:
I'm allowing the users to set the room number already, so index might not be that great...